optuna_dashboard/ts/components/PreferentialTrials.tsx

```diff
@@ -12,41 +24,119 @@ const PreferentialTrial: FC<{
   const theme = useTheme()
   const action = actionCreator()
   const trialWidth = 500
+  const trialHeight = 300
+  const [detailShown, setDetailShown] = useState(false)
 
   if (trial == undefined) {
-    return <Box width={trialWidth}></Box>
+    return (
+      <Box
+        sx={{
+          width: trialWidth,
+          minHeight: trialHeight,
+          margin: theme.spacing(2),
+        }}
+      />
+    )
   }
 
   return (
-    <Box sx={{ width: trialWidth, padding: theme.spacing(2, 2, 0, 2) }}>
-      <Typography
-        variant="h4"
-        sx={{
-          marginBottom: theme.spacing(2),
-          fontWeight: theme.typography.fontWeightBold,
-        }}
-      >
-        Trial {trial.number} (trial_id={trial.trial_id})
-      </Typography>
-      <Button
-        variant="outlined"
+    <Card
+      sx={{
+        width: trialWidth,
+        minHeight: trialHeight,
+        position: "relative",
+        margin: theme.spacing(2),
+        padding: 0,
+      }}
+    >
+      <CardActions>
```

Comment:
Not a strong opinion, but how about placing a header text like this?

```suggestion
      <CardActions>
         <Typography variant="h5">
           Trial {trial.number}
         </Typography>
```

<img width="1873" alt="Screenshot 2023-08-28 16 41 13" src="https://github.com/optuna/optuna-dashboard/assets/5564044/3fb6bb06-6cd5-4fd4-8f01-d1bb9c38e28e">